webserver/script/LoadForecasting/LoadForecastingRunner.py

```diff
@@ -45,16 +46,19 @@ def lf_runner(
     total[:, 1] = model.load_segments['Residential L2']['Load']
     total[:, 2] = model.load_segments['Residential MUD']['Load']
     total[:, 3] = model.load_segments['Work']['Load']
-    total[:, 4] = model.load_segments['Fast']['Load'][np.arange(0, 1440, 15)]
+    total[:, 4] = model.load_segments['Fast']['Load'][np.arange(0, 1440, 1)]
     total[:, 5] = model.load_segments['Public L2']['Load']
     total[:, 6] = np.sum(total, axis=1)
     total_df = pd.DataFrame(data=total, columns=['Residential L1', 'Residential L2', 'Residential MUD', 'Work', 'Fast', 'Public L2', 'Total'])
 
-    # pd.DataFrame(model.sampled_loads_dict).to_csv('s3://script.forecast.inputsoutputs/Control_Paper_Scenarios/'+name+'_uncontrolled_load.csv')
+    # save csv of dataframe for later CBA processing
+    path = Path(__file__).parent.resolve()
+    parent_path = path.parent
+    pd.DataFrame(model.sampled_loads_dict).to_csv(str(parent_path)+'/costbenefitanalysis/preprocessing_loadprofiles/inputs/weekdays/BaseCase_2025_weekday_' + county + '_county_uncontrolled_load.csv')
```

Comment:
Added this line to save the dataframe as a csv to run through the preprocessing module.

Reply:
Path leads to a non-existent directory

Reply:
It is in the gitignore. I will fix it so the directory is committed

Reply:
Thanks

Reply:
fixed - thanks derin!